runner.go

```diff
@@ -538,3 +540,22 @@ func expandStringSlice(slice []string) []string {
 	}
 	return result
 }
+
+// writeRunID writes the run ID to a file if the flag is set
+func writeRunID(c cliContext, runStatus *rainforest.RunStatus) error {
+	filePath := c.String("save-run-id")
+	if filePath == "" {
+		return nil
+	}
+
+	file, err := os.Create(filePath)
+	defer file.Close()
+
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	} else {
+		file.WriteString(strconv.Itoa(runStatus.ID))
```

Comment:
Is this new line terminated? IMO it should be.

Reply:
It is not because the files we currently generate in the Orb/Action aren't, but from testing, a newline shouldn't cause isses, so I'll add one 👍🏼